suite-common/formatters/src/tests/cryptoToFiatAmountFormatter.test.ts

```diff
@@ -0,0 +1,348 @@
+                        ltc: 244.6,
+                        usd: 21257,
+                        mmk: 44788652,
+                        sats: 99982276,
+                        krw: 26256741,
+                        chf: 19455.27,
+                        myr: 91821,
+                        xau: 11.12,
+                        inr: 1729531,
+                        xag: 879.02,
+                        eth: 13.450276,
+                        dot: 3574,
+                        aed: 78078,
+                        ils: 72026,
+                        idr: 320556248,
+                        link: 3138,
+                        xrp: 54801,
+                        bits: 999823,
+                        gbp: 17233.92,
+                        bdt: 2209515,
+                        sek: 219396,
+                        twd: 644955,
+                        eur: 19592.47,
+                        nok: 209346,
+                        cny: 143732,
+                        ngn: 9821314,
+                        btc: 1,
+                        dkk: 145753,
+                        sgd: 28029,
+                        rub: 1460700,
+                        sar: 79837,
+                        kwd: 6494.7,
+                        huf: 7754189,
+                        yfi: 2.969723,
+                    },
+                    ts: 1674034245,
+                },
+            ],
+        },
+        symbol: 'btc',
+    },
+];
+
+describe('CryptoAmountFormatter', () => {
+    describe('formats correctly', () => {
+        const CryptoToFiatAmountFormatter = prepareCryptoToFiatAmountFormatter({
+            coins,
+            locale: 'en',
+            bitcoinAmountUnit: 0,
+            fiatCurrency: 'usd',
+            // @ts-expect-error
+            intl: {
+                formatNumber: new Intl.NumberFormat('en').format,
+            },
+        });
+
+        it('converts correctly', () => {
+            expect(
+                CryptoToFiatAmountFormatter.format('20638700284758254', {
+                    network: 'eth',
+                }),
+            ).toBe('31.96');
+        });
+
+        it('converts with symbol', () => {
+            expect(
+                CryptoToFiatAmountFormatter.format('500743', {
+                    network: 'btc',
+                }),
+            ).toBe('113.5');
+        });
+    });
+
+    describe('czech intl format', () => {
+        const CryptoToFiatAmountFormatter = prepareCryptoToFiatAmountFormatter({
+            coins,
+            locale: 'czk',
+            bitcoinAmountUnit: 0,
+            fiatCurrency: 'usd',
+            // @ts-expect-error
+            intl: {
+                formatNumber: new Intl.NumberFormat('pt').format,
+            },
+        });
+
+        it('converts correctly', () => {
+            expect(
+                CryptoToFiatAmountFormatter.format('20638700284758254', {
+                    network: 'eth',
+                }),
+            ).toBe('31,96');
```

Comment:
are you sure this is correct? seems like very small value for that much ETH :D

Reply:
that's because that value is not "formatted" :D So it's like sats, if ethereum has sats